src/reducers/index.js

```diff
@@ -3,10 +3,12 @@ import { combineReducers } from 'redux';
 // Reducers
 import eventsReducer from './events-reducer';
 import overlayReducer from './overlay-reducer';
+import popupReducer from './popup-reducer';
 
 // Combine Reducers
 const reducers = combineReducers({
   eventsState: eventsReducer,
   overlayState: overlayReducer,
+  popupState: popupReducer,
```

Comment:
Can it be just events, overlay, popup instead of name+State?

Reply:
fixed